documentation/book/proc-configuring-kafka-mirror-maker-tls.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-mirror-maker-tls.adoc
+
+[id='proc-configuring-kafka-mirror-maker-tls-{context}']
+= Configuring TLS encryption in Kafka Mirror Maker
+
+.Prerequisites
+
+* An {ProductPlatformName} cluster
+* A running Cluster Operator
+
+.Procedure
+
+Because Kafka Mirror Maker connects to two Kafka clusters (source and target), it is possible to configure TLS for one of them or both.
```

Comment:
Rewrite as suggested in the "Configuring SCRAM-SHA-512 authentication in Kafka Mirror Maker" module.